libs/ui/src/components/Explorer/Explorer.tsx

```diff
@@ -87,7 +92,8 @@ export const Explorer: FunctionComponent<IExplorerProps> = ({
         fulltextSearch: view.fulltextSearch,
         pagination: noPagination ? null : {limit: view.pageSize, offset: view.pageSize * (currentPage - 1)},
         sorts: view.sort,
-        filters: view.filters
+        filters: view.filters,
+        skip: viewSettingsLoading
```

Comment:
**question:** Est ce que tu penses possibles de récupérer les 2 appels en 1 coup ? https://www.apollographql.com/blog/using-apollo-link-to-handle-dependent-queries#option-1-update-the-graphql-server-to-adhere-to-frontend-needs 

Je me dis que c'est un peu la promesse de GraphQL, qu'est ce qui nous empêche de le faire ?

Reply:
Dans ce cas précis, je ne pense pas qu'on puisse que les 2 requêtes soit vraiment dépendantes. Au premier affichage, oui, on doit aller chercher des données sur les paramètres par défaut. Mais ensuite la requêtes data vit sa vie sans avoir besoin de l'autre.
Je n'ai jamais testé les requêtes dépendantes, ça vaudrait peut-être le coup de creuser un peu.